setup

```diff
@@ -1,9 +1,7 @@
 #!/usr/bin/env python3
-
 import pip
 from shutil import copyfile
 
-
```

Comment:
[Following PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines) I'm against removing this line. It's a subjective interpretation, but I think the function calls are the main part of this file – ie I'd suggest a two lines separation as PEP8 suggests for top-level function/class definitions.